### PR TITLE
dashboard: Reduce dependency to cockpit 138

### PR DESCRIPTION
**Pull request by dperpeet**
The dashboard and ssh technically require 138 plus patches,
but downstream version 138 is already patched.

In Fedora we've had plenty of updates since then and we don't
want to run into this issue with each rebase.

- [ ] confirm this works downstream

### CODE REVIEW

**Comment by dperpeet** [comment]
This doesn't work, the base package still only provides 138.

**Comment by dperpeet** [comment]
Do we really break any Cockpit installations if we just change this to 138 on master?

**Comment by martinpitt** [comment]
Not "break" if you just keep updating, but the calculated dependencies are insufficient then. And this especially applies to RHEL which has this rather wild mix of binary versions. For example, if you still have 138-5 from base installed, -dashboard 141 from extras just depending on ">= 138" would not suffice, things would break; the correct dependency for RHEL really is ">= 138-6".

min-base-version seems to work when changing the manifest to "138-6":
```
$ tools/min-base-version  dashboard
138-6
```

**Comment by martinpitt** [comment]
Argh! I just saw that our -dashboard package does not use `required_base` and thus the above mechanism in the rhel-* branches, so indeed "138-6" wouldn't help us. Furthermore, someone seems to have ripped out the required_base dependency calculation entirely from master, which is just plain wrong. This needs some git archeology and restoring.

I misunderstood this -- this isn't about package dependencies being too high (>= 138.x) at all, but cockpit's own module version check not accepting the "138-6". So for the RHEL binary version intermix we have to reduce the manifest version to "138", as cockpit's version checks don't know about package revisions. However, in fact the package dependency of cockpit-dashboard is too *weak* right now -- in the rhel* branches we need to explicitly add a Requires to >= 138-6.

So, in summary:

 * Bite the bullet and accept this manifest change, even though it's conceptually wrong for master, but there's no other way for RHEL 7.3.
 * Add an explicit Requires cockpit-bridge >= 138-6 in the rhel-7.3 branch for -dashboard.
 * Restore required_base RPM dependency calculation on master (unrelated to this PR).

**Comment by martinpitt** [comment]
Edit: required_base is still in master, sorry; PEBCAK. I edited the comment above.
